SimPEG/Directives.py

```diff
@@ -845,13 +832,11 @@ def startIRLS(self):
                 print("eps_p: " + str(reg.eps_p) +
                       " eps_q: " + str(reg.eps_q))
 
-    @property
-    def _angleScale(self):
+    def angleScale(self):
```

Comment:
do you want users accessing this? if not (in most use cases), I would suggest keeping the underscore in front